### ISSUE TITLE
Acetabular erosions

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#6716]
**Preferred term label:**
Acetabular erosions

**Synonyms**

**Definition (free text, please give PubMed ID)**
Erosion (loss of substance) of the acetabular subchondral cortical bone. The acetabulum is the concave surface that meets with the head of the femur, forming the hip joint.

**Parent term (use hpo.jax.org/app)**
Abnormality of the acetabulum HP:0003170

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM:601492 
https://pubmed.ncbi.nlm.nih.gov/8793927/
1/1

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
